routes.js

```diff
@@ -0,0 +1,48 @@
+const express = require('express');
+const app = express();
+const port = 5000;
+
+const data = [
+  {
+    "NAME": "Janhavi",
+    "ROLL NO.": 11111,
+    "PHONE NO.": "911122334",
+    "MEME": "Meme",
+    "PHOTOGRAPH": "Photo",
+    "REASON": "Reason",
+    "HOUR SLEPT": 2
+  },
+  
+];
+
+app.use(express.json());
+
+app.get('/', (req, res) => {
+    res.json(data)
+});
+
+app.post('/', (req, res) => {
```

Comment:
It's good practice to validate the incoming request body in your POST and PUT routes. This will ensure that you're not inserting or updating your data with invalid data. [important]